packages/server/utils/OpenAIServerManager.ts

```diff
@@ -43,107 +41,101 @@ class OpenAIServerManager {
     }
   }
 
-  // setting the minimal max_tokens limit increases the speed of the request and reduces the tokens used
-  estimateOutputTokens(reflectionsText: string[]) {
-    const avgGroupNameLength = 10 // assume an average group name length of 10 tokens
-    const totalLengthInTokens = reflectionsText.reduce(
-      (total, reflectionText) => total + Math.ceil(reflectionText.length / AVG_CHARS_PER_TOKEN),
-      0
-    )
-    const avgReflectionsPerGroup = 3 // assume each group will have around 3 reflections
-    const numGroups = Math.ceil(reflectionsText.length / avgReflectionsPerGroup)
-
-    const outputTokens = Math.round(numGroups * avgGroupNameLength + totalLengthInTokens)
+  async generateThemes(reflectionsText: string[]) {
+    if (!this.openAIApi) return null
+    const suggestedThemeCountMin = Math.floor(reflectionsText.length / 5)
+    const suggestedThemeCountMax = Math.floor(reflectionsText.length / 4)
+    const prompt = `Create a short list of common themes given the following reflections: ${reflectionsText.join(
+      ', '
+    )}. Each theme should be no longer than a few words. There should be roughly ${suggestedThemeCountMin} or ${suggestedThemeCountMax} themes. Return the themes as a comma-separated list.`
```

Comment:
I'm requesting roughly this many themes because otherwise it tends to create way too many

Reply:
-1 can you add this as a comment please, so it's not forgotten?